server/stycoon/model_static_data_ship_classes_value.go

```diff
@@ -24,6 +24,8 @@ type StaticDataShipClassesValue struct {
 	Life int64 `json:"life,omitempty"`
 
 	Damage int64 `json:"damage,omitempty"`
+
+	Price int64 `json:"price,omitempty"`
```

Comment:
Why use `omitempty` everywhere?
The bots won't be all in Go so the fields will be actually missing after parsing.
Wouldn't it be easier for bots/players to condition on `damage == 0` than on missing field?
Because e.g. for shipyard you get a condition "shipyard == true" instead of "shipyard is in keys && shipyard == true".

Does this even follow the API spec?

Sorry for not so related comment to this PR.

Reply:
I was equally triggered as you are :D

The API works like this - when it is not required it is omitempty. 

I wanted to make the price required, but mothership does not have this field ... I will create an issue for this so we can discuss it.